core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -266,32 +257,16 @@ private static void fix(Attributes atts, List<PluginWrapper.Dependency> optional
         if (jenkinsVersion==null)
             jenkinsVersion = atts.getValue("Hudson-Version");
         
-        optionalDependencies.addAll(getImpliedDependencies(pluginName, jenkinsVersion));
+        optionalDependencies.addAll(DetachedPluginsUtil.getImpliedDependencies(pluginName, jenkinsVersion));
     }
-    
+
     /**
-     * Returns all the plugin dependencies that are implicit based on a particular Jenkins version
-     * @since 2.0
+     * @see DetachedPluginsUtil#getImpliedDependencies(String, String)
      */
+    @Deprecated // since TODO
```

Comment:
We usually use `FIXME` for planned `@since` tags (easier to search for), but anyway I think there is no need to give a date/version on this. Just replace `@see` with a `@deprecated` tag.